cmd/cli/commands/orders.go

```diff
@@ -13,22 +13,22 @@ var (
 )
 
 func init() {
-	marketSearchCmd.PersistentFlags().Uint64Var(&ordersSearchLimit, "limit", 10, "Orders count to show")
+	ordersListCmd.PersistentFlags().Uint64Var(&ordersSearchLimit, "limit", 10, "Orders count to show")
 
-	marketRootCmd.AddCommand(
-		marketSearchCmd,
-		marketShowCmd,
-		marketCreateCmd,
-		marketCancelCmd,
+	ordersRootCmd.AddCommand(
+		ordersListCmd,
+		orderStatusCmd,
+		orderCreateCmd,
+		orderCancelCmd,
 	)
 }
 
-var marketRootCmd = &cobra.Command{
-	Use:   "market",
-	Short: "Interact with Marketplace",
+var ordersRootCmd = &cobra.Command{
+	Use:   "orders",
```

Comment:
Why in plural?

Reply:
because everything is in plural, I like plural.
cli orders list
cli orders show 42
Plural, moth***r, do you speak it?